third_party/blink/renderer/platform/media/web_media_player_impl.cc

```diff
@@ -2838,6 +2823,14 @@ std::unique_ptr<media::Renderer> WebMediaPlayerImpl::CreateRenderer(
   media_metrics_provider_->SetRendererType(renderer_type_);
   media_log_->SetProperty<MediaLogProperty::kRendererName>(renderer_type_);
 
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  LOG(INFO) << "Renderer Type is " << GetRendererName(renderer_type_) << ".";
```

Comment:
This is already logged in Media DevTools in l.2824, is this extra LOG() needed?

Reply:
We may not always use DevTools for media. This can ensure we can have this in adb bugreport.